core/arch/arm/include/crypto/ghash-ce-core.h

```diff
@@ -9,7 +9,10 @@
 #include <inttypes.h>
 
 struct internal_ghash_key {
-	uint64_t k[2];
+	uint64_t h[2];
```

Comment:
lots of those `uint64_t xx[2];` in this P-R.
Maybe introduce `TEE_AES_BLOCK_SIZE_U64` a `TEE_AES_BLOCK_SIZE / sizeof(uint64_t)` ?

Reply:
I have considered the same, but so far I haven't found anything that beats a plain `2`.